app/main/views/digital_outcomes_and_specialists.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from app import data_api_client
+
+from flask import abort, render_template
+from ...helpers.buyers_helpers import get_framework_and_lot
+from ...main import main
+
+
+@main.route('/buyers/frameworks/<framework_slug>/requirements/user-research-studios', methods=['GET'])
+def studios_start_page(framework_slug):
+    framework = data_api_client.get_framework(framework_slug)['frameworks']
```

Comment:
`user-research-studios` are tied to a specific framework, I think we still need to check that the framework has the lot